src/components/inputs/CheckboxButton/CheckboxButton.js

```diff
@@ -1,5 +1,60 @@
-import styled from 'styled-components'
-import BaseCheckboxButton from './CheckboxButton.base'
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled, { css } from 'styled-components'
+import ErrorMessage from 'SRC/components/inputs/ErrorMessage'
+
+const BaseCheckboxButton = ({className, input, value, error, children, ...props}) => {
+  return (
+    <div className={className}>
+      <input id={value} type='checkbox' value={value} {...props} />
+      <label htmlFor={value}>
+        {children}
+      </label>
+      {error &&
+        <ErrorMessage>{error}</ErrorMessage>
+      }
+    </div>
+  )
+}
+
+const handleShapeStyles = shape => {
+  switch (shape) {
```

Comment:
Perhaps we could simplify this by using a hash? Something like:

```js
const squareStyle = css`
  height: 12rem;
  width: 12rem;
`

const rectangleStyle = css`
  height: 6.4rem;
  width: 100%;
`

const shapeVariants = {
  square: squareStyle,
  rectangle: rectangleStyle,
  ...
}

function handleShapeStyles(shape) {
  return shapeVariants[shape]
}

Reply:
I like that idea! Switch/case's always feel awkward to me